lib/chat_api_web/views/customer_view.ex

```diff
@@ -1,11 +1,22 @@
 defmodule ChatApiWeb.CustomerView do
   use ChatApiWeb, :view
-  alias ChatApiWeb.{CustomerView, TagView}
+  alias ChatApiWeb.{CustomerView, TagView, CSVHelpers}
+
+  @customer_csv_ordered_fields ~w(id name email created_atm updated_at)a ++
+                                 ~w(first_seen last_seen phone external_id) ++
+                                 ~w(host pathname current_url browser) ++
+                                 ~w(os ip metadata time_zone)a
```

Comment:
let's just remove `metadata` for now (since it's a `map`/`json` field, and might break things)